### ISSUE TITLE
Progress bar/spinner animations with NoopAnimationsModule

### ISSUE THREAD

**Issue by bennyboer** [angular/material2#15713]
#### What is the expected behavior?
I expect progress bar and spinner animations to still work even when the `NoopAnimationsModule` is imported.

#### What is the current behavior?
Animations are commonly disabled using the `NoopAnimationsModule` in order to gain additional performance especially on mobile devices. Still progress spinners (indeterminate) and bars do not make sense without animation.
As mentioned under the [pull request](https://github.com/angular/material2/pull/10881) which disabled the animations, it might make sense to import the `NoopAnimationsModule` for e2e and screenshot tests. But wouldn't it make sense to rather introduce a hidden switch for that especially for those tests instead of turning off animations completely?

#### What are the steps to reproduce?
> Use `mat-spinner` with NoopAnimationsModule imported.

See [StackBlitz example](https://stackblitz.com/edit/angular-material2-issue-u1wk46)

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular & Angular Material 7

**Comment by milie-clarisoft** [comment]
You can use the injection token MAT_PROGRESS_SPINNER_DEFAULT_OPTIONS, providing it with different configuration in the module where you need it (or globally in AppModule), e.g.:
```
@NgModule({
    ...
    providers: [
        {
            provide: MAT_PROGRESS_SPINNER_DEFAULT_OPTIONS,
            useValue: {
                _forceAnimations: true
            }
        }
    ],
    ...
})
export class AppModule {
}
```

Check the docs for all the available configuration params: https://material.angular.io/components/progress-spinner/api#interfaces

**Comment by maic333** [comment]
You can use the injection token MAT_PROGRESS_SPINNER_DEFAULT_OPTIONS, providing it with different configuration in the module where you need it (or globally in AppModule), e.g.:
```
@NgModule({
    ...
    providers: [
        {
            provide: MAT_PROGRESS_SPINNER_DEFAULT_OPTIONS,
            useValue: {
                _forceAnimations: true
            }
        }
    ],
    ...
})
export class AppModule {
}
```

Check the docs for all the available configuration params: https://material.angular.io/components/progress-spinner/api#interfaces

**Comment by bennyboer** [comment]
@maic333 Yes, that's the solution I am currently using, though it is unavailable for progress bars.

**Comment by vinod-amt** [comment]
> @maic333 Yes, that's the solution I am currently using, though it is unavailable for progress bars.

yes, not available for progress bars. any work around?

**Comment by alperozhan** [comment]
actually @maic333's solution works with @angular/material 11.0.0 in my case.

**Comment by k290** [comment]
This would be a useful fix as we use NoopAnimations to disable animations only in some places in our software

**Comment by crisbeto** [comment]
Closing since `_forceAnimations` is the way to do it.
